src/interfaces/Order.ts

```diff
@@ -0,0 +1,15 @@
+export interface Order{
+    /*
+    * Notice the Order interface is NOT a one for one with the Order table in our api.
```

Comment:
Wait I'm confused, this looks pretty one for one to me. 🤔

Reply:
Yea when i had to rethink it i changed it without removing those comments